src/main/scala/temple/generate/target/openapi/OpenAPIGenerator.scala

```diff
@@ -0,0 +1,21 @@
+package temple.generate.target.openapi
+
+import temple.generate.target.openapi.OpenAPIType._
+import io.circe.syntax._
+
+import scala.collection.immutable.ListMap
+
+object OpenAPIGenerator {
+
+  /** Create a Response representation for an error */
+  private[openapi] def generateError(description: String, code: Int, example: String): Response =
+    Response(
+      description,
+      ListMap(
```

Comment:
Did we work out that ListMap maintains dict order?

Reply:
Yes